source/agora/common/Serializer.d

```diff
@@ -87,6 +87,20 @@ public void serializePart (T) (scope const auto ref T record, scope SerializeDg
         serializePart(v, dg);
 }
 
+/// Write the serialization data to a pre-allocated buffer (useful for tests)
+package(agora) ubyte[] serializeToBuffer (T) (scope const auto ref T record, scope ubyte[] buffer)
```

Comment:
If it's for tests only, shouldn't it be `version (unittest)`?

Reply:
Good point